samples/README.md

```diff
@@ -1,4 +1,10 @@
 # Cordova HockeySDK sample
+This repository contains a sample app that allows for easy testing of the SDK and its functionality.
+
+### Prerequisites
+1) Condova should be installed locally on your machine.
+2) Hockey application should be created on portal before testing.
```

Comment:
"Please create a new app in the HockeyApp portal before you start testing with the sample app"